configuration/types.go

```diff
@@ -351,6 +351,8 @@ type Configuration struct {
 	OnlineURL string `json:"online_url"`
 
 	// DataDirectory is a folder used to store logs and any data used to perform validation.
+	// The path can be absolute or relative to the directory containing the rosetta-cli
+	// binary.
```

Comment:
Feels like this comment could be a bit misleading. It's either absolute or relative to where the binary is being executed instead of the path to the binary